src/ambassadors/species.ts

```diff
@@ -0,0 +1,457 @@
+    },
+  },
+  blueAndGoldMacaw: {
+    name: "Blue and Gold Macaw",
+    scientificName: "Ara ararauna",
+    iucn: { id: 22685539, status: "LC/decreasing" },
+    native: {
+      text: "South America (tropical rainforests)",
+      source: "https://en.wikipedia.org/wiki/Blue-and-yellow_macaw",
+    },
+    lifespan: {
+      wild: 50,
+      captivity: 70,
+      source:
+        "https://animaldiversity.org/accounts/Ara_ararauna/#lifespan_longevity",
+    },
+  },
+  catalinaMacaw: {
+    name: "Catalina Macaw",
+    scientificName: "Ara ararauna x Ara macao",
+    iucn: { id: null, status: "NE" },
+    native: {
+      text: "Usually bred in captivity",
+      source: "https://en.wikipedia.org/wiki/Catalina_macaw",
+    },
+    lifespan: {
+      captivity: 70,
+      source:
+        "https://animaldiversity.org/accounts/Ara_macao/#lifespan_longevity",
+    },
+  },
+  africanGrey: {
+    name: "African Grey",
+    scientificName: "Psittacus erithacus",
+    iucn: { id: 22724813, status: "EN/decreasing" },
+    native: {
+      text: "Equatorial Africa (dense forests)",
+      source: "https://en.wikipedia.org/wiki/African_grey_parrot",
+    },
+    lifespan: {
+      wild: 23,
+      captivity: 60,
+      source: "https://en.wikipedia.org/wiki/African_grey_parrot",
+    },
+  },
+  blueFrontedAmazon: {
+    name: "Blue-fronted Amazon",
+    scientificName: "Amazona aestiva",
+    iucn: { id: 22686332, status: "NT/decreasing" },
+    native: {
+      text: "South America (tropical rainforests)",
+      source: "https://en.wikipedia.org/wiki/Blue-fronted_amazon",
+    },
+    lifespan: {
+      captivity: 60,
+      source: "https://animaldiversity.org/accounts/Amazona_aestiva/",
+    },
+  },
+  americanCrow: {
+    name: "American Crow",
+    scientificName: "Corvus brachyrhynchos",
+    iucn: { id: 22705990, status: "LC/increasing" },
+    native: {
+      text: "North America (forests)",
+      source: "https://en.wikipedia.org/wiki/American_crow",
+    },
+    lifespan: {
+      wild: 7,
+      captivity: 30,
+      source: "https://en.wikipedia.org/wiki/American_crow",
+    },
+  },
+  oliveEggerChicken: {
+    name: "Olive Egger Chicken",
+    scientificName: "Gallus gallus domesticus",
+    iucn: { id: null, status: "NE" },
+    native: {
+      text: "North America (domesticated)",
+      source: "https://en.wikipedia.org/wiki/Chicken",
+    },
+    lifespan: {
+      captivity: 8,
+      source: "https://animal-world.com/olive-egger-chickens/",
+    },
+  },
+  ameraucanaChicken: {
+    name: "Ameraucana Chicken",
+    scientificName: "Gallus gallus domesticus",
+    iucn: { id: null, status: "NE" },
+    native: {
+      text: "North America (domesticated)",
+      source: "https://en.wikipedia.org/wiki/Chicken",
+    },
+    lifespan: {
+      captivity: 8,
+      source: "https://animal-world.com/olive-egger-chickens/",
+    },
+  },
+  halfBrahmaHalfSaipanChicken: {
```

Comment:
Not sure if there's a better way to represent this or not

Reply:
I think this is probably fine -- my only other thought would be to have it be `chickenHalfBrahmaHalfSaipan` for better sorting and whatnot, but definitely not a strong opinion

Reply:
Hmm good point, I think it would make sense to do the same for the other species as well. Like `ameraucanaChicken` -> `chickenAmeraucana`, `redAngusBeefCow` -> `cowRedAngusBeef`

It does make the names a little awkward in my opinion though, wdyt?

Reply:
Yeh I would imagine we'd want to apply it to everything if we did, and agree it does make things a bit awkward -- I don't have a strong opinion here so whatever your gut says is best is fine 👍 

If we stick with what we have currently, but you want a bit more "organization" as it were, perhaps a comment for each species group? E.g. `// Chickens`, similar to how we demarcate active/retired in the ambassadors core data?